### PR TITLE
Use bold for solution explorer and italics for file name

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/desktop-wpf/migration/differences-from-net-framework.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ There are a few features of .NET Framework that .NET Core doesn't support. For m
 
 If your .NET Framework app lists its NuGet dependencies in a *packages.config* file, migrate to the [`<PackageReference>`](/nuget/consume-packages/package-references-in-project-files) format.
 
-01. In Visual Studio, find the *Solution Explorer* pane.
-01. With your WPF app, right-click **packages.config** > **Migrate packages.config to PackageReference**.
+01. In Visual Studio, find the **Solution Explorer** pane.
+01. Within your WPF project, right-click *packages.config* > **Migrate packages.config to PackageReference...**.
```
Comment (mairaw) [inline docs/desktop-wpf/migration/differences-from-net-framework.md]: Our guidelines specifically say we don't show the ... (I'll get the exact text)

```suggestion
01. Within your WPF project, right-click *packages.config* > **Migrate packages.config to PackageReference**.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. Left you a small suggestion.

File: docs/desktop-wpf/migration/differences-from-net-framework.md (hunk 1 above)
Comment (mairaw) [reply docs/desktop-wpf/migration/differences-from-net-framework.md]: Here it is: https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions 

If an option label ends with a colon or an ellipsis, don't include that end punctuation in instructions.

Comment (Youssef1313) [reply docs/desktop-wpf/migration/differences-from-net-framework.md]: @mairaw, Thanks! Didn't know that.

**Review by mairaw — APPROVED** [review]
Thank you @Youssef1313.

**Review by Thraka — APPROVED** [review]
LGTM Thanks!

**Pull request merged**